### DISCUSSION TITLE
virtink dashboard

### DISCUSSION THREAD

**Discussion by vishalkadam47** [smartxworks/virtink#96] [Q&A]
is there a friendly Web User Interface like 
https://kubevirt-manager.io/

**Comment by fengye87** [comment]
We've previously tried to implement an Octant plugin, but sadly Octant has been archived by VMware. So the plan for now is that we would still like to implement an web UI plugin of Virtink, for our own's plugin based K8s web dashboard (which may open source later). But the dashboard itself is a WIP (although we are pushing it rapidly), so Virtink's web UI would not be ready quite soon.

So the short answer is, there's no web UI for now, but we have a plan for it and is working on it hardly.

**Reply by vishalkadam47** [reply]
As of now I'm using kubevirt-manager, But I wish to use Virtink web UI when ever its available, if possible please add Authentication RBAC and SSO.

**Reply by fengye87** [reply]
👌 Authentication, RBAC and SSO (with LDAP and OAuth2) will all be on the plate :)

**Comment by vishalkadam47** [comment]
kubevirt is still not in my plate, really very complicated to deploy 
virtink way was easy i was able to deploy a vm on kubernetes can you guide me 
how do i deploy windows 10 or 11 vm

**Reply by fengye87** [reply]
Well, I've never tried to deploy Win 10/11 VM on Virtink, but I do have deployed a Win server 2012 on KubeVirt successfully. So I can only give you some tips based on my previous experience deploying Win server on KubeVirt:

1. You'll need a qcow2 image of Win 10/11. Since Virtink doesn't provide VNC access to VM, installing from ISO is not possible.
2. You should install tools like cloudbase-init in your qcow2 image to support cloud-init booting of your VM
3. You should access your VM via windows remote desktop

**Comment by vishalkadam47** [comment]
is there any example virtink.yaml using qcow2 Win server -  https://cloudbase.it/windows-cloud-images/#download
so that I can try one. 
or will you be able to help me please with the yaml
